scripts/common/initialize-dependencies-java-tool.sh

```diff
@@ -9,11 +9,12 @@ if [ "$BUILD_TOOL" == "maven" ]; then
     # TODO update to use build tool if specified
```

Comment:
Can we remove the commented out code if it's not needed? On this line and below

Reply:
@timrbula Agreed, done.